docs/guides/development/mcp/build-mcp-server.mdx

```diff
@@ -67,47 +67,19 @@ sdk: nextjs, expressjs
     - [`@vercel/mcp-adapter`](https://github.com/vercel/mcp-adapter): A utility library that simplifies building an MCP server by handling the core protocol logic for you. It also includes an authentication wrapper that allows you to plug in your own token validation - in this case, using Clerk's OAuth tokens.
     - [`@clerk/mcp-tools`](https://github.com/clerk/mcp-tools): A helper library built on top of the [MCP TypeScript SDK](https://github.com/modelcontextprotocol/typescript-sdk) used to connect Clerk OAuth with MCP easily.
 
-    <CodeBlockTabs options={["npm", "yarn", "pnpm", "bun"]}>
-      ```bash {{ filename: 'terminal' }}
-      npm install @vercel/mcp-adapter @clerk/mcp-tools
-      ```
-
-      ```bash {{ filename: 'terminal' }}
-      yarn add @vercel/mcp-adapter @clerk/mcp-tools
-      ```
-
-      ```bash {{ filename: 'terminal' }}
-      pnpm add @vercel/mcp-adapter @clerk/mcp-tools
-      ```
-
-      ```bash {{ filename: 'terminal' }}
-      bun add @vercel/mcp-adapter @clerk/mcp-tools
-      ```
-    </CodeBlockTabs>
+    ```npm
+    npm install @vercel/mcp-adapter @clerk/mcp-tools
```

Comment:
`@vercel/mcp-adapter` is now `mcp-handler`.

See README → https://www.npmjs.com/package/@vercel/mcp-adapter

This will likely require more changes than just this code block. How would you like to handle this? Separate PR?

Reply:
Created [DOCS-11163](https://linear.app/clerk/issue/DOCS-11163/rename-vercelmcp-adapter-to-mcp-handler) to address this separately.